code/src/reducers/tasks.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const tasks = createSlice({
+  name: 'tasks',
+  initialState: { items: [
+    { id: 0, text: 'Add a todo!'},
+  ] },
+  reducers: {
+    addItem: (state, action) => {
+      const addItems = [...state.items,action.payload]
+      state.items = addItems
+    },
+    removeItem: (state, action) => {
+      const filterItems = state.items.filter((item) => item.id !== action.payload.id)
+      state.items = filterItems
+    }
+  }
+})
```

Comment:
Nice job Max! One action (and property of the items) missing though - toggling whether it's a checked item or not. If you check the first todo, create a new one and then remove the checked one - then the second todo gets checked. Also - don't forget to prevent the user from submitting empty todos :)